gnome/spice-gtk/Portfile

```diff
@@ -5,11 +5,11 @@ PortGroup           active_variants 1.1
 PortGroup           meson 1.0
 
 name                spice-gtk
-version             0.39
+version             0.41
 revision            1
```

Comment:
When updating the version, you'll also want to reset `revision` to zero.

Reply:
Thanks, done.